code/game/machinery/tcomms/nttc.dm

```diff
@@ -112,13 +129,17 @@
 		"Journalist" = "srvradio",
 		"Librarian" = "srvradio",
 		"Mime" = "srvradio",
+		"Sous Chef" = "srvradio",
+		"Sous-Chef" = "srvradio",
```

Comment:
This isnt an ingame title and I dont want to encourage everyone making their custom HoP assignments part of the master repo.

Reply:
Fair, it can be removed if you strongly object.

Though Sous-Chef is probably the most common custom service job out there, and while I personally never played a "Sous-Chef" in all my 2000+ hours on Paradise, I've seen enough two-man kitchen set-ups with that title that I assumed this addition would be welcome to that part of the playerbase.